lib/puppet-lint/plugins/check_whitespace/right_lbrace_whitespace.rb

```diff
@@ -0,0 +1,25 @@
+# Public: Check the manifest tokens for whitespace after the left bracket that is missing and record a warning for each instance found.
+# #
+# https://docs.puppet.com/guides/style_guide.html#spacing-indentation-and-whitespace
+PuppetLint.new_check(:right_lbrace_whitespace) do
+  def check
+    tokens.each do |token|
+      next if token.next_token.nil?
+      if token.type == :LBRACE 
```

Comment:
@rnelson0 The reason I did not test for multiple here is because I found that including multiple whitespace in the test added a lot more control flow requirements to the fix logic than was palatable...additionally it made atomizing each actual problem longer and more complicated. That is why the multiple whitespace is its own test.